src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -290,6 +295,12 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
   lazy val stringLiteral: Parser[StringLiteral] =
     stringLit ^^ StringLiteral
 
+  lazy val numberLiteral: Parser[NumberLiteral] =
+    numericLit ^^ {s =>
```

Comment:
Coding style: missing space between `{` and `s`.

Reply:
Oops. Fixed in https://github.com/sjrd/scala-js-ts-importer/pull/47/commits/8bf944f1ce9b26122978dad991a3ac5891f43e66